todo-mvc/src/index.html

```diff
@@ -34,7 +34,9 @@
 			</app-projector>
 		</section>
 		<footer class="info">
-			<p>Double-click to edit a todo</p>
+			<app-projector>
+				<app-widget id="footer-instructions" data-factory="src/widgets/createTextWidget"></app-widget>
```

Comment:
is it okay if we do this programmatically for now? we've temporarily stopped using `data-factory` with a mid until we can resolve issues around building with webpack.